admin/projectsConfig.mjs

```diff
@@ -0,0 +1,55 @@
+const GRAPHQL_FIELD_STUDY = 'Study';
+const GRAPHQL_FIELD_PARTICIPANT = 'Participant';
+
+// Reminder: by arranger standards, project id must be lowered case.
+const PROJECTS_IDS = {
+    migration_test_qa: 'migration_test_qa',
```

Comment:
Not sure i understant why the project id is in the code. Should that not be in a env variable or somthing? for each now project we have to re-release kf-arranger-api? Maybe im missing something, but i think it should come from here: https://github.com/d3b-center/d3b-deployment-config/tree/master/application_configuration/kf-strides/us-east-1